backend/src/daemon/fwupdmgr.rs

```diff
@@ -0,0 +1,128 @@
+use std::process::Command;
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug)]
+#[allow(non_snake_case)]
+struct Devices {
+    pub Devices: Vec<Device>,
```

Comment:
The idiomatic way to serialize fields with this sort of naming is to use `#[serde(rename_all = "PascalCase")]`.